terraform/lib/dependabot/terraform/version.rb

```diff
@@ -30,7 +30,9 @@ class Version < Dependabot::Version
       end
       def initialize(version)
         @version_string = T.let(version.to_s, String)
-        super
+        @version_string = @version_string.gsub(/^v/, "")
```

Comment:
@BlakeBosin could you please add a link to the 
```suggestion
       # See terraform specs: https://developer.hashicorp.com/terraform/registry/modules/publish#releasing-new-versions
        @version_string = @version_string.gsub(/^v/, "")
```

Reply:
tooke your suggestion on the comment.

i did run the test without passing @version_string but it will be required because when super gets called without a vars being passed it assumes "version" would be passed but we modified "version" input to make the version without "v" if it was passed in.  See failure from test below.  So because we are modifying version passed in and maintaining the original value that was passed in "@version_string" replaces "version".

`Failures:

  1) Dependabot::Terraform::Version#to_s allow optional v on release 
     Failure/Error: super

     ArgumentError:
       Malformed version number string v1.0.0`

Reply:
@abdulapopoola - let me know if you would like other changes.  ill try to watch my email over the next few days.  thank you for your current responses on this.

Reply:
I talked with @abdulapopoola and sounds like it's resolved from his point of view.